app/containers/Network/containers/BulkEditAccessPoints/index.js

```diff
@@ -0,0 +1,330 @@
+import React, { useContext, useState, useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { Alert, notification } from 'antd';
+import { useQuery, useLazyQuery, useMutation } from '@apollo/react-hooks';
+import { BulkEditAccessPoints, Loading } from '@tip-wlan/wlan-cloud-ui-library';
+import { IS_2DOT4GHZ, IS_5GHZL, IS_5GHZU } from 'constants/index.js';
+import { GET_ALL_LOCATIONS, FILTER_EQUIPMENT_BULK_EDIT_APS } from 'graphql/queries';
+import { UPDATE_EQUIPMENT_BULK } from 'graphql/mutations';
+
+import UserContext from 'contexts/UserContext';
+import styles from './index.module.scss';
+
+const renderTableCell = tabCell => {
+  if (Array.isArray(tabCell)) {
+    return (
+      <div className={styles.tabColumn}>
+        {tabCell.map(i => (
+          <span>{i}</span>
+        ))}
+      </div>
+    );
+  }
+  return <span>{tabCell}</span>;
+};
+const accessPointsChannelTableColumns = [
+  { title: 'NAME', dataIndex: 'name', key: 'name', render: renderTableCell },
+  {
+    title: 'CHANNEL',
+    dataIndex: 'channel',
+    key: 'channel',
+    editable: true,
+    render: renderTableCell,
+  },
+  {
+    title: 'CELL SIZE',
+    dataIndex: 'cellSize',
+    key: 'cellSize',
+    editable: true,
+    render: renderTableCell,
+  },
+  {
+    title: 'PROB RESPONSE THRESHOLD',
+    dataIndex: 'probeResponseThreshold',
+    key: 'probeResponseThreshold',
+    editable: true,
+    render: renderTableCell,
+  },
+  {
+    title: 'CLIENT DISCONNECT THRESHOLD',
+    dataIndex: 'clientDisconnectThreshold',
+    key: 'clientDisconnectThreshold',
+    editable: true,
+    render: renderTableCell,
+  },
+  {
+    title: 'SNR (% DROP)',
+    dataIndex: 'snrDrop',
+    key: 'snrDrop',
+    editable: true,
+    render: renderTableCell,
+  },
+  {
+    title: 'MIN LOAD',
+    dataIndex: 'minLoad',
+    key: 'minLoad',
+    editable: true,
+    render: renderTableCell,
+  },
+];
+
+const BulkEditAPs = () => {
+  const { customerId } = useContext(UserContext);
+  const [selectedLocationIds, setSelectedLocationIds] = useState([]);
+  const { id } = useParams();
+  const { data } = useQuery(GET_ALL_LOCATIONS, {
+    variables: { customerId },
+  });
```

Comment:
you don't need to get locations pass the selected locations from Network as prop